src/style.css

```diff
@@ -1,3 +1,89 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+html {
+  --link-hover: #00acdc;
+  --nav-link-height: 60px;
+
+  font-family: Roboto, Arial, "Helvetica Neue", Helvetica, sans-serif;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
 }
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+[class*="__container"] {
+  max-width: 1130px;
+  padding: 0 15px;
+  margin: 0 auto;
+}
+
+@media (min-width: 1024px) {
```

Comment:
no needs to use media queries. it's enough just use full parent width and padding: 0 50px; for your header